scripts/benchmarks.py

```diff
@@ -0,0 +1,320 @@
+# coding=utf-8
+#
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis-python
+#
+# Most of this work is copyright (C) 2013-2017 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at http://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+# pylint: skip-file
+
+from __future__ import division, print_function, absolute_import
+
+from collections import OrderedDict
+import attr
+import hypothesis.strategies as st
+from hypothesis import settings
+from hypothesis.internal.conjecture.engine import ConjectureRunner
+from hypothesis.internal.conjecture.data import StopTest
+from hypothesis.errors import UnsatisfiedAssumption
+from scipy.stats import ttest_ind
+import click
+import os
+import random
+import hashlib
+import numpy as np
+import sys
+from collections import Counter
+
+ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Same comment here about script path being changed as in #537, although I'm not sure if we want to add a hard dependency on a `git` binary in here. Thoughts?